tests/local.rs

```diff
@@ -121,13 +141,13 @@ mod firefox {
     #[test]
     #[serial]
     fn new_window_switch_test() {
-        local_tester!(new_window_switch, "firefox")
+        tester!(new_window_switch, "firefox")
```

Comment:
Why did these change to be non-local?

Reply:
They don't need to be - they aren't using the local server for anything.
Ideally, the local server is used to test the recent Iframe additions, and possibly migrate some remote tests to, but the windowing tests can be done mostly without needing to access a given page.

Reply:
I actually wonder if ultimately we want to move almost _all_ the tests to be local, so that they won't accidentally break just because some website changes, but that's a change we can make later with the groundwork you've now laid down :)

Reply:
Filed as https://github.com/jonhoo/fantoccini/issues/76